Sources/Fullscreen/Stories/Models/StorySlideViewModel.swift

```diff
@@ -5,16 +5,25 @@ public struct StorySlideViewModel {
     public let title: String?
     public let detailText: String?
     public let price: String?
+    public let read: Bool?
+    public let notificationId: Int?
+    public let adId: Int?
 
     public init(
         imageUrl: String?,
         title: String?,
         detailText: String?,
-        price: String?
+        price: String?,
+        read: Bool?,
+        notificationId: Int?,
+        adId: Int?
```

Comment:
Just a heads up: this is a breaking change and not a patch, since the newly added parameters doesn't have a default value.

Reply:
Good point! Added default values